src/MagnumPlugins/UfbxImporter/Test/UfbxImporterTest.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019,
+                2020, 2021, 2022 Vladimír Vondruš <mosra@centrum.cz>
+    Copyright © 2022 Samuli Raivio <bqqbarbhg@gmail.com>
+
+    Permission is hereby granted, free of charge, to any person obtaining a
+    copy of this software and associated documentation files (the "Software"),
+    to deal in the Software without restriction, including without limitation
+    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+    and/or sell copies of the Software, and to permit persons to whom the
+    Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included
+    in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+*/
+
+#include <sstream>
+#include <Corrade/Containers/Optional.h>
+#include <Corrade/Containers/ArrayView.h>
```

Comment:
```cpp
#include <Corrade/Containers/StringView.h>
```

to fix the `nondeprecated` build. A lot of compatibility StringView includes got added some time ago to aid porting away from STL strings, and the build w/o deprecated features doesn't have those anymore. I have to clean that up, sorry for the mess.

Easiest for you will probably be if you compile Corrade with the `CORRADE_BUILD_DEPRECATED` CMake option disabled, and Magnum with `MAGNUM_BUILD_DEPRECATED` disabled, that way you'll also have the build significantly faster.

Reply:
Thanks for the tip! I copied the base from `StlImporterTest.cpp` but I assume one of the headers there indirectly included `StringView.h` which made it work. Fixed it with an explicit include now!